TelegramClient/src/main/java/com/praisesystem/telegramclient/README.md

```diff
@@ -0,0 +1,15 @@
+Basic Telegram Praise Bot (work in progress)
+
+Receceives praise in the format:
+
+/praise @user1, @user2, @user3 for doing something awesome
+
+and sends it to the backend in the following format
+
+TIME: "[Unix Timecode]" GIVER: "@sender" RECEIVER: "@user1" CONTENT:  "doing something awesome"
```

Comment:
I agree with leaving it as TO DO. I wanted to use java.time which uses ISO8601, but I think that would involve making changes to the gradle source compatibility and I don't feel confident to do that in case it breaks something with the rest of the project.